src/kibana/components/agg_response/geo_json/_tooltip_formatter.js

```diff
@@ -0,0 +1,32 @@
+define(function (require) {
+  return function TileMapTooltipFormatter($compile, $rootScope) {
+    var $ = require('jquery');
+
+    var $tooltipScope = $rootScope.$new();
+    var $tooltip = $(require('text!components/agg_response/geo_json/_tooltip.html'));
+    $compile($tooltip)($tooltipScope);
+
+    return function tooltipFormatter(feature) {
+      if (!feature) return '';
+
+      var details = $tooltipScope.details = [];
+
+      var lat = feature.geometry.coordinates[1];
+      var lng = feature.geometry.coordinates[0];
+
+      var metric = {
+        label: feature.properties.valueLabel,
+        value: feature.properties.count
+      };
+      var location = {
+        label: 'Center',
+        value: lat.toFixed(4) + ', ' + lng.toFixed(4)
```

Comment:
This is fine for now, but should be using the field formatter when we go to fix that on the other tilemap modes.